zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -31,11 +31,13 @@
 
 #include <stdio.h>           /* fprintf */
```

Comment:
`stdio` is needed for debug,
but also for `FINISH_WITH_ERR` macros.

See comments below